content/en/docs/concepts/overview/working-with-objects/common-labels.md

```diff
@@ -170,4 +170,4 @@ metadata:
 
 With the MySQL `StatefulSet` and `Service` you'll notice information about both MySQL and Wordpress, the broader application, are included.
 
-{{% /capture %}}
\ No newline at end of file
+{{% /capture %}}
```

Comment:
Not sure what changed here. I just edited the instance values with the github UI.